### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by Tiandishihua** [YuelangX/Gaussian-Head-Avatar#22]
Can I use RGB video from a monocular camera for training? We look forward to hearing from you, thank you.

**Comment by YuelangX** [comment]
Yes, you can follow the instruction to preprocess your monocular video for training. Actually, I tested on a set of monocular data. The rendering results from the side view look a lot worse. 

https://github.com/YuelangX/Gaussian-Head-Avatar/assets/132422760/656912dc-98eb-4438-9e95-fc51eefa7a8d



https://github.com/YuelangX/Gaussian-Head-Avatar/assets/132422760/3b2182aa-d081-40eb-be2b-f2211a682f73

**Comment by jeb0813** [comment]
Hi @YuelangX , for monocular data, how do you get the camera params(extrinsic, intrinsic).

**Comment by YuelangX** [comment]
You could refer to https://github.com/YuelangX/Multiview-3DMM-Fitting. The camera params are manually specified.

**Comment by NikoBele1** [comment]
@YuelangX I am trying to use this repo with my monocular data not from Nersemble dataset. 

Do you know a way to generate the camera params for the dataset structure required in your repo Multiview-3DMM-Fitting ? As in, frame1.jpg - camera1.npz, frame2.jpg - camera2.npz, etc.

**Comment by YuelangX** [comment]
@NikoBele1 , you can run this script https://github.com/YuelangX/Multiview-3DMM-Fitting/blob/6f64ee1bb1545ce3f3a83706f07f857b0f28ab5d/preprocess/preprocess_monocular_video.py

**Comment by NikoBele1** [comment]
oh my bad, i thought the camera params where required, i misread that the repository creates them.

**Comment by NikoBele1** [comment]
@YuelangX sorry to disturb you again, but i have a question. I am currently training on a monocular video (539 frames, preprocessed with Multiview-3DMM-Fitting) and am at 600 epochs. I used checkpoint 600 to do self-reenactment on my monocular video and this is the result i get:

![intermediate_GHA_1_720](https://github.com/YuelangX/Gaussian-Head-Avatar/assets/32575893/57c8c538-28b6-48cf-9ecc-7d8af2411dc5)

This does not look like Gaussian Splatting, i would expect to see larger splats everywhere. Problems i know: The camera params are not correct, since they are manual in Multiview-3DMM-Fitting/preprocess_monocular_video.py. And i noticed that there is no lowres landmark in myDataset/mySubject/landmarks/*/. after preprocess.

Do you have an idea what could be wrong here? I know training is not finished, but i would expect to see a different intermediate result.

**Comment by YuelangX** [comment]
@NikoBele1 This seems strange, is the result during training also like this?

**Comment by YuelangX** [comment]
@NikoBele1 This seems strange, are the results during training also like this?

**Comment by NikoBele1** [comment]
@YuelangX thanks for replying. Tracking looks fine, besides the strange cropping. Images have 2k resolution and i don't know what could have led to the cropping.

![413900](https://github.com/YuelangX/Gaussian-Head-Avatar/assets/32575893/cd78a1a6-96a5-4923-adfa-32cdf7fbd9a9)

**Comment by jryebread** [comment]
@NikoBele1 hi did you figure out the fix to your issue for custom character?

**Comment by NikoBele1** [comment]
@jryebread what issue do you have? the cropping or the other weird reenactment?
